### PR TITLE
Changelog added for OPCUA improvement DM-4085

### PR DESCRIPTION
**Pull request by PestusAtSAG**
Improvement regards Nordex Issue: https://cumulocity.atlassian.net/browse/DM-4085

### CODE REVIEW

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/opcua-10-20-96-improved-agent-issue-process-mappings-for-device-values.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+date: 
+title: Improve OPC UA operation ProcessMappingsForDeviceValues
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-96-improved-agent-issue-process-mappings-for-device-values.md]:
```suggestion
title: Improved OPC UA operation ProcessMappingsForDeviceValues
```

File: content/change-logs/device-management/opcua-10-20-96-improved-agent-issue-process-mappings-for-device-values.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-Tf05_KQ-B
+    label: OPC UA
+build_artifact:
+  - value: tc-MLn0oFRX-
+    label: opcua
+ticket: DM-4085
+version: 10.20.96.0
+---
+
+In previous versions, the operation `c8y_ua_command_ProcessMappingsForDeviceValues` would fail entirely if any single mapping failed. The new behavior allows the operation to continue reading nodes even if some cannot be read due to bad status codes. Although the operation will still be marked as failed, all readable nodes will be mapped. Additionally, the result and failure messages have been improved to indicate which nodes failed and which were successful.
```
Comment (BeateRixen) [inline content/change-logs/device-management/opcua-10-20-96-improved-agent-issue-process-mappings-for-device-values.md]:
```suggestion
In previous versions, the operation `c8y_ua_command_ProcessMappingsForDeviceValues` failed entirely if any single mapping failed. The new behavior allows the operation to continue reading nodes even if some nodes cannot be read due to bad status codes. Although the operation will still be marked as failed, all readable nodes will be mapped. Additionally, the result and failure messages have been improved to indicate which nodes failed and which were successful.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Oezge-Akin Can this one be merged?

**Comment by Oezge-Akin** [comment]
> @Oezge-Akin Can this one be merged?

yes. merging

**Pull request merged**